webapp/channels/src/plugins/exported_create_post.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import AdvancedCreateComment from 'components/advanced_create_comment';
+
+import type {PostDraft} from 'types/store/draft';
+
+type Props = {
+    placeholder?: string;
+    onSubmit: (draft: PostDraft) => void;
+}
+
+const ExportedCreatePost = ({placeholder, onSubmit}: Props) => {
+    const Component = AdvancedCreateComment as any;
```

Comment:
Not the best, but if the plan from @larkox is to clean up these components. We can fix these types then.